context.go

```diff
@@ -58,27 +60,36 @@ type Context struct {
 	srtpSessionAuth    hash.Hash
 	srtpSessionAuthTag []byte
 	srtpBlock          cipher.Block
+	srtpReplayDetector replaydetector.ReplayDetector
 
 	srtcpSessionKey     []byte
 	srtcpSessionSalt    []byte
 	srtcpSessionAuth    hash.Hash
 	srtcpSessionAuthTag []byte
 	srtcpIndex          uint32
 	srtcpBlock          cipher.Block
+	srtcpReplayDetector replaydetector.ReplayDetector
 }
 
 // CreateContext creates a new SRTP Context
-func CreateContext(masterKey, masterSalt []byte, profile ProtectionProfile) (c *Context, err error) {
+func CreateContext(masterKey, masterSalt []byte, profile ProtectionProfile, opts ...ContextOption) (c *Context, err error) {
```

Comment:
I think this might be too generic/not self documenting. 

What do you think of adding this as either a method or a mandatory argument? I don't know if a developer consuming this package would understand what `ContextOption` means.

Reply:
`ContextOption` is following functional options pattern, so it basically generic (we can put any option changing `Context` private members) and not only for replay protection.

IMO, it's better to avoid breaking API at now.
There is another API breaking backlog. It would be better to break API at one time.

In my experience, enabling SRTP replay protection at default may cause breakage on receiving stream from independently running command like gst-launch, since such command randomly select initial sequence number. Reinitialized stream is discarded by replay protection.